ci/update-requirements.py

```diff
@@ -76,6 +76,10 @@ def pc(target, *sources):
         'demos/python_demos/requirements.txt', openvino_dir / 'python/requirements.txt')
     pc('ci/requirements-downloader.txt',
         'tools/downloader/requirements.in')
+    pc('ci/requirements-quantization.txt',
+        'tools/accuracy_checker/setup.py',
+        openvino_dir / 'deployment_tools/tools/post_training_optimization_toolkit/setup.py',
+        openvino_dir / 'deployment_tools/model_optimizer/requirements.txt')
```

Comment:
I suspect that POT doesn't actually need all of the frameworks that MO supports. I suggest that you try to use one of MO's `requirements_<framework>.txt` files instead (preferably the one with the least amount of dependencies in it) and see if POT still works.

Reply:
Done. (Works with kaldi requirements; [logs](http://213.221.44.203/job/omz/job/precommit/job/downloader_ubuntu18/466/console))